README.md

```diff
@@ -85,7 +85,7 @@ Please see the [Changelog](CHANGELOG.md) for more details about a given release.
 
 ## Konnect
 Kong Inc. offers commercial subscriptions that enhance the Kong API Gateway in a variety of ways. Customers of Kong's [Konnect](https://konghq.com/kong-konnect/) subscription take advantage of additional gateway functionality, commercial support, and access to Kong's managed (SaaS) control plane platform. The Konnect platform features include real-time analytics, a service catalog, developer portals, and so much more! [Get started](https://konghq.com/get-started/) with Konnect.
-
+Say Hello to Kongs
```

Comment:
May I know why we need this change? 😄

Reply:
Because it looks friendly to new contributors like me 😊